Libraries/Text/RCTTextView.m

```diff
@@ -57,6 +58,8 @@ - (instancetype)initWithBridge:(RCTBridge *)bridge
 
     _backedTextInput.textInputDelegate = self;
 
+    self.font = self.fontAttributes.font;
```

Comment:
Why do we need this?

Reply:
It ensures that `_backedTextInput's` font gets initialized to `RCTFontAttributes's` default font. This is not necessarily the same as `_backedTextInput's` default font.